src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -1,20 +1,31 @@
-If you choose [`sentry-cli`](/product/cli/) as the method to upload your source files, we assume you already know how to generate source maps within your build tool chain.
+If you choose [`sentry-cli`](/product/cli/) as the method to upload your artifacts, we assume you already know how to generate source maps within your build toolchain.
 
-Sentry uses [releases](/product/releases/) to match the correct uploaded source maps to your events. You can use `sentry-cli` to create releases and upload source maps to Sentry.
+Sentry has two ways of matching the correct uploaded source maps to your events:
 
-## 1. Update SDK Options
+1. The first is to use Debug IDs, which allow Sentry to automatically associate minified sources with their respective source maps without needing to create a release. _This method is the preferred way of uploading artifacts._
```

Comment:
```suggestion
1. Using Debug IDs allows Sentry to automatically associate minified sources with their respective source maps without needing to create a release. _This is the preferred method of uploading artifacts._
```
Since there are just two methods and we mention it above, saying the first and the second in addition to the numbers feels repetitive.